camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -138,7 +138,7 @@ def run(self):
 class CalibrationNode:
     def __init__(self, boards, service_check = True, synchronizer = message_filters.TimeSynchronizer, flags = 0,
                 fisheye_flags = 0, pattern=Patterns.Chessboard, camera_name='', checkerboard_flags = 0,
-                max_chessboard_speed = -1, queue_size = 1, no_gui=False, no_mono_if_stereo=False):
+                max_chessboard_speed = -1, queue_size = 1, no_gui=False, no_mono_if_stereo=False, mask = False):
```

Comment:
no whitespaces around = here
whitespaces are added when you are using typing, but without, you don't add them.
optionally, you can fix the old values, but at least you should not continue the pattern :)